src/LibraryManager.Vsix/UI/InstallDialog.xaml.cs

```diff
@@ -145,9 +145,11 @@ private void CloseDialog(bool res)
             Close();
         }
 
-        private void OnLoaded(object sender, RoutedEventArgs e)
+        private async void OnLoadedAsync(object sender, RoutedEventArgs e)
         {
-            ViewModel = new InstallDialogViewModel(Dispatcher, _libraryCommandService, _configFileName, _deps, _fullPath, CloseDialog);
+            EnvDTE.Project project = await VsHelpers.GetProjectOfSelectedItemAsync();
```

Comment:
>GetProjectOfSelectedItemAsync [](start = 53, length = 29)

Instead of doing this here, pass it into the constructor from InstallLibraryCommand.

Reply:
Will do

---
In reply to: [202841135](https://github.com/aspnet/LibraryManager/pull/317#discussion_r202841135) [](ancestors = 202841135)